Scoop/Controller/Home/HomeViewController.swift

```diff
@@ -169,24 +171,39 @@ class HomeViewController: UIViewController {
                     
                     if let date = dateFormatterGet.date(from: ride.departureDatetime) {
                         rideCopy.departureDatetime = dateFormatterPrint.string(from: date)
+                    }
+                    
+                    /// Separate a user's rides into those that are active and those that are pending
+                    var activeRidesIDs = []
+                    for ride in NetworkManager.shared.currentUser.rides {
+                        activeRidesIDs.append(ride.id)
+                    }
```

Comment:
If the order does not matter, then I recommend using the `map` higher order function instead of a for-in loop here:
`activeRidesIDs = NetworkManager.shared.currentUser.rides.map { $0.id }`
Alternatively, you could use the `forEach` higher order function if order matters.

Reply:
wow i learn smth new everyday, thanks vin :>